toolkit/tools/internal/packlist/packagelist.go

```diff
@@ -4,44 +4,29 @@
 package packagelist
 
 import (
-	"bufio"
-	"fmt"
-	"os"
 	"strings"
 
 	"github.com/microsoft/azurelinux/toolkit/tools/internal/timestamp"
 )
 
-// ParsePackageListFile will parse a list of packages to pack or parse, if one is specified.
-// Duplicate list entries in the file will be removed.
-// If no path is specified, nil will be returned.
-// A set of [package name] -> [true] will be returned.
-func ParsePackageListFile(packageListFile string) (packageList map[string]bool, err error) {
+// ParsePackageList will parse a string of packages.
+// Duplicate entries in the string will be removed.
+// If empty string is given, nil will be returned.
+// Else, a map of [package name] -> [true] will be returned.
+func ParsePackageList(packages string) (packageMap map[string]bool, err error) {
 	timestamp.StartEvent("parse list", nil)
 	defer timestamp.StopEvent(nil)
 
-	if packageListFile == "" {
+	if len(strings.TrimSpace(packages)) == 0 {
```

Comment:
Would it be better to call to `strings.Fields()` first, then check the `len(list)` after? Can skip this special handling.